javaslang/src/test/java/javaslang/collection/euler/Euler17Test.java

```diff
@@ -99,7 +99,7 @@ default int letterCount(Seq<Integer> range) {
             1_000, "thousand",
             1_000_000, "million",
             1_000_000_000, "billion"
-    );
+    ).grouped(2).toSortedMap(pair -> Tuple.of((Integer) pair.get(0), (String) pair.get(1)));
```

Comment:
For type safety and better documentation via tests, maybe we should use `.merge()` instead, i.e.
```java
static final Map<Integer, String> LENGTHS = TreeMap.of(
        1, "one",
        2, "two",
        3, "three",
        4, "four",
        5, "five",
        6, "six",
        7, "seven",
        8, "eight",
        9, "nine",
        10, "ten"
).merge(TreeMap.of(
        11, "eleven",
        12, "twelve",
        13, "thirteen",
        14, "fourteen",
        15, "fifteen",
        16, "sixteen",
        17, "seventeen",
        18, "eighteen",
        19, "nineteen",
        20, "twenty"
)).merge(TreeMap.of(
        30, "thirty",
        40, "forty",
        50, "fifty",
        60, "sixty",
        70, "seventy",
        80, "eighty",
        90, "ninety",
        100, "hundred",
        1_000, "thousand",
        1_000_000, "million",
        1_000_000_000, "billion"
));
```
(though this doesn't seem to compile ... @danieldietrich, why is this working if I extract the parts and merge them in a method, but not inline?)

Maybe we could extend the `merge` to vararg also, so that larger maps can be created in a safe way also :)